src/components/Entry.vue

```diff
@@ -143,13 +187,34 @@ export default {
       loadingSpinnerSvg: assets.loadingSpinnerSvg,
       deleteSvg: assets.deleteSvg,
       addSvg: assets.addSvg,
+      birthdate: { month: "", day: "", year: "" },
+      itemList: [
+        {
+          id: "month",
+          text: "Month",
+          valuesList: Array.from({ length: 12 }, (_, i) => i + 1),
+        },
+        {
+          id: "day",
+          text: "Day",
+          valuesList: Array.from({ length: 31 }, (_, i) => i + 1),
+        },
+        {
+          id: "year",
+          text: "Year",
+          valuesList: Array.from({ length: 30 }, (_, i) => i + 1989).reverse(),
+        },
+      ],
     };
   },
   created() {
     /** The user type is set as soon as component is created */
     this.userType = this.groupData.userType;
   },
   computed: {
+    isExtraInputValidationRequired() {
+      return this.authType.split(",").length > 1;
```

Comment:
whats this `.split(",").length > 1` part?

Reply:
This tells us if there are more than one input that needs to be validated.